compiler/passes/convert-typed-uast.cpp

```diff
@@ -5764,6 +5764,12 @@ bool TConverter::enter(const Conditional* node, RV& rv) {
       attachSymbolVisibility(ifVar, ifVarSym);
     } else {
       cond = convertExpr(node->condition(), rv, &qtCond);
+      // TODO: need to resolve _cond_test
```

Comment:
Is `_cond_test` always concrete? It feels like it's maybe one of those things that we might just want to have a widget for to resolve once.

Reply:
It's generic and does some reflection in certain cases, unfortunately.